rclpy/rclpy/node.py

```diff
@@ -1091,6 +1091,31 @@ def remove_waitable(self, waitable: Waitable) -> None:
         self.__waitables.remove(waitable)
         self._wake_executor()
 
+    def resolve_topic_name(self, topic: str, *, only_expand: bool = False) -> str:
+        """
+        Return a topic name expanded and remapped.
+
+        :param only_expand: if `True`, remapping rules won't be applied.
```

Comment:
We should document the `topic` argument.

Reply:
```suggestion
        :param topic: topic name to be expanded and remapped.
        :param only_expand: if `True`, remapping rules won't be applied.
```